tests/test_manipulation.py

```diff
@@ -448,6 +455,520 @@ def test_no_copy(self):
         assert_array_equal(b, a)
 
 
+class TestPad:
```

Comment:
Will it make sense to decouple tests for `pad` into a separate `test_pad.py` or `test_arraypad.py` scope?

Reply:
created `test_arraypad.py` for `dpnp.pad` tests.